app/views/layouts/application.html.erb

```diff
@@ -13,4 +13,8 @@
   <body>
     <%= yield %>
   </body>
+
+  <% if flash[:alert].present? %>
+  <p class="my_alert"><%= flash[:alert] %></p>
+<% end %>
```

Comment:
In terms of user experience, you typically put the (important) error messages at the top of the page so that they don't get pushed down by the rest of the page content and missed by the user. In terms of code, you can't have `<p>` tags outside of the body, so this is invalid HTML.